objectivec/GPBWellKnownTypes.m

```diff
@@ -115,3 +115,40 @@ - (void)setTimeIntervalSince1970:(NSTimeInterval)timeIntervalSince1970 {
 }
 
 @end
+
+NSString *const GPBTypeGoogleApisComPrefix = @"type.googleapis.com/";
+
+@implementation GPBAny (GBPWellKnownTypes)
+
+- (NSString*)getAnyTypeName {
```

Comment:
There are two places to change type url. One is from the accessor of type_url, the other is from initWithMessage.
For the first one, it needs to modify the code in GPBMessage specifically for any. This seems complicated.
For the second, we can guarantee type_url is valid in our implementation. User have no way to change type_url. So we don't need the assert.